Glossary.md

```diff
@@ -37,7 +37,16 @@ Description is a language-specific descriptive phrase for an [Entity](#entity).
 
 ## Entity
 
-Entities are the top level concepts of Wikibase's data model. [Items](#item) and [Properties](#property) are the core Entity types of Wikibase. Other types can be added through [Wikibase extensions](#wikibase-extension), such as [Lexemes](https://www.mediawiki.org/wiki/Extension:WikibaseLexeme/Data_Model#Lexeme).
+Entities are the top level concepts of Wikibase's data model. [Items](#item) and [Properties](#property) are the core Entity types(#entity-types) of Wikibase. Other types can be added through [Wikibase extensions](#wikibase-extension), such as [Lexemes](https://www.mediawiki.org/wiki/Extension:WikibaseLexeme/Data_Model#Lexeme).
+
+## [Entity Sources](https://doc.wikimedia.org/Wikibase/master/php/md_docs_topics_entitysources.html)
+
+The source of an entity type. Only a single entity source is allowed per entity type on a given repo.
+
+## [Entity Types](https://doc.wikimedia.org/Wikibase/master/php/md_docs_topics_entitytypes.html)
+
+Entities as defined by Wikibase have a unique identifier and a type.
```

Comment:
This sounds more like the definition of entities which is already done above.
Entities do indeed and a type and ID.

I would say that an entity type right now has a defined ID format? And JSON structure?

Entity types can have even more though, per the entity type wiring that we allow to be set.
There is a minimum set of services etc that need to be set, these include serializer / serializer things, Id factory etc.

Reply:
Updated the definition.